samples/asgardeo-react-app/src/pages/IssuerClaimValidationFailuer.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Let's use the new license header.

https://github.com/wso2/identity-apps/blob/master/.eslintrc.js#L1

```suggestion
 * Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
fixed in commit https://github.com/asgardeo/asgardeo-auth-react-sdk/commit/2ded725d7c96494fedec39cb54bdd58556460ce6